### PR TITLE
Added extra logging

### PR DESCRIPTION
**Pull request by cagiti**
Added extra logging to help debug #4104

### CODE REVIEW

File: pkg/jx/cmd/opts/common.go [hunk 1]
```diff
@@ -259,7 +262,9 @@ func (o *CommonOptions) JXClient() (versioned.Interface, string, error) {
 	if o.factory == nil {
 		return nil, "", errors.New("command factory is not initialized")
 	}
+
 	if o.jxClient == nil {
+		log.Infof("Creating new JX Client")
```
Comment (ccojocar) [inline pkg/jx/cmd/opts/common.go]: Shouldn't we use `Debug` for this? I can get extra verbose if you intent to use it permanently.

**Comment by hferentschik** [comment]
@cagiti, is this still something we need after pr #4108 got merged? Should this be closed now?

**Pull request closed without merging**